cdap-docs/examples-manual/source/examples/user-profiles.rst

```diff
@@ -0,0 +1,245 @@
+.. meta::
```

Comment:
No, I modified it so that you run it twice: once as Row, and see the errors, and then a second time as Column, and see the errors disappear.